src/runner/centralized_kb_hook.cpp

```diff
@@ -0,0 +1,109 @@
+#include "pch.h"
+#include "centralized_kb_hook.h"
+
+namespace CentralizedKeyboardHook
+{
+    struct HotkeyDescriptor
+    {
+        Hotkey hotkey;
+        std::wstring moduleName;
+        std::function<bool()> action;
+
+        bool operator<(const HotkeyDescriptor& other) const 
+        {
+            return hotkey < other.hotkey;
+        };
+
+    };
+
+    std::multiset<HotkeyDescriptor> hotkeyDescriptors;
```

Comment:
if we don't care about descriptors ordering, perhaps we could use `std::unordered_multiset` instead?

Reply:
We could, but then I'd need to write a hasher for `Hotkey`, and I'm too lazy and it's not worth it :)

Reply:
same amount of lines as `operator<` :P